src/styles/sidebar/components/PaginationNavigation.scss

```diff
@@ -0,0 +1,50 @@
+@use '../../mixins/buttons';
```

Comment:
<img width="721" alt="prev-next-asymmetry" src="https://user-images.githubusercontent.com/2458/107951983-0df47e00-6f91-11eb-820d-31a55924dc19.png">

I see an asymmetry between the "Prev" and "Next" buttons. The position and size of each are different. Above is a screenshot with the `:hover` pseudo-class enabled for both in dev tools.

Reply:
This layout has been giving me some enduring grief! After pounding on it for a while today, I ended up refactoring it using CSS-grid, and was able to accomplish what I was trying to do with minimal fuss. The challenge is that I want the space used by the three components here (previous button, page number buttons, next button) to remain constant, even when the previous or next button isn't present. That's challenging to accomplish with `flexbox` as things are, well, flexing. It's a little easier to be explicit with `grid`.

I bet someone who is actually _good_ at `flexbox` would have no trouble with this.

I'll push the style refactoring here as a separate, new commit for easier review (it's not a huge change but I don't want it getting lost in the large diff for yah).

Reply:
Thanks for the update. If CSS grid makes the preferred layout easier to achieve then its fine to use it.